ui/src/shared/components/DatabaseDropdown.js

```diff
@@ -43,12 +43,18 @@ class DatabaseDropdown extends Component {
     const proxy = source.links.proxy
     try {
       const {data} = await showDatabases(proxy)
-      const {databases} = showDatabasesParser(data)
+      const {databases, errors} = showDatabasesParser(data)
+      if (errors.length > 0) {
+        throw errors[0] // only one error can come back from this, but it's returned as an array
```

Comment:
ah yes, i forget now what i was thinking then, but i see that when you throw, it's caught by the `catch` block. 👍